src/metabase/driver/googleanalytics/query_processor.clj

```diff
@@ -4,6 +4,7 @@
             [clojure.tools.reader.edn :as edn]
             [medley.core :as m]
             [metabase.query-processor.util :as qputil]
+            [metabase.query-processor.middleware.expand :as ql]
```

Comment:
I'm doing this manually right now, an emacs fn would be much better. Did you get that function here? https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-clean-ns